src/App.tsx

```diff
@@ -1,24 +1,233 @@
-/* eslint-disable max-len */
 /* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
+import React, {
+  useState,
+  useEffect,
+  useCallback,
+  useMemo,
+} from 'react';
+
+import {
+  addTodo,
+  deleteTodo,
+  getTodos,
+  updateTodo,
+} from './api/todos';
+import { Todo } from './types/Todo';
 import { UserWarning } from './UserWarning';
+import { FilterBy } from './types/FilterBy';
+import { TempTodo } from './types/TempTodo';
+import { TodoList } from './components/TodoList';
+import { NewTodo } from './components/NewTodo';
+import { Filter } from './components/Filter';
+import { TodoWarning } from './components/TodoWarning';
+import { ErrorMessage } from './types/ErrorMessage';
 
-const USER_ID = 0;
+const USER_ID = 6401;
 
 export const App: React.FC = () => {
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [isError, setIsError] = useState(false);
+  const [filterBy, setFilterBy] = useState<FilterBy>(FilterBy.all);
+  const [title, setTitle] = useState('');
+  const [disabledInput, setDisabledInput] = useState(false);
+  const [tempTodo, setTempTodo] = useState<TempTodo | null>(null);
+  const [errorMessage, setErrorMessage]
+    = useState<ErrorMessage>(ErrorMessage.none);
+  const [loadingTodoIds, setLoadingTodoIds] = useState([0]);
+
+  const getTodosFromServer = useCallback(
+    async () => {
+      try {
+        const todosList = await getTodos(USER_ID);
+
+        setTodos(todosList);
+      } catch (error) {
+        setIsError(true);
+        setErrorMessage(ErrorMessage.loadTodo);
+      }
+    }, [todos],
+  );
+
+  useEffect(() => {
+    getTodosFromServer();
+  }, []);
+
+  const allTodosCompleted = useMemo(() => (
+    todos.every(todo => todo.completed)), []);
```

Comment:
Probably toggling all todos to complete doesn't work properly cause of missing here `todos` dependency. 
And also styles of the toggle button don't work properly due to this.